### ISSUE TITLE
SliderRow not disabled

### ISSUE THREAD

**Issue by alldritt** [xmartlabs/Eureka#942]
- Environment: Eureka (master), Xcode 8.2.1 and iOS 10.2.1.

When I try and disable a SliderRow, the control is not disabled.  I'm disabling the row like this:

```swift
            let disabled = Condition.function(["disabled"], { (form) -> Bool in
                return true
            })

            row.disabled = disabled
            row.evaluateDisabled()
```

This technique works with Name, Stepper and Switch rows.

**Comment by bhushanb02** [comment]
@alldritt, I have created a pull request to fix this issue. Till then you can add `slider.isEnabled = !row.isDisabled` line in `update` function of `SliderCell` class.

**Comment by m-revetria** [comment]
Closing this, looks like it was solved in #1064
